### ISSUE TITLE
The gameplay loop

### ISSUE THREAD

**Issue by Steve-D-Eckles** [AndrewScholly/The-Darkness#6]
The 'game' here is to just type 'fire blaster' over and over again and then you get automatically sent through a big series of rooms rolling random numbers until you automatically win.

You should probably add more choices, and RNG causing you to have to make the same choice over and over isn't very compelling. Randomness is a cool thing to implement, but should probably not be the core of the experience.

**Comment by AndrewScholly** [comment]
Yeah, that was just a basic showing of the idea. I likely will move classes and stuff around some to make it better and randomized for different encounters for each room, or more rooms.
